src/publicodes-state/hooks/useDisposableEngine/index.ts

```diff
@@ -41,9 +42,19 @@ export default function useDisposableEngine({ rules, situation }: Props) {
     engine.setSituation(newSituation, { keepPreviousSituation: true })
   }
 
+  const getSubcategories = useCallback(
+    (dottedName: DottedName) =>
+      (getSomme(safeGetRule(dottedName)?.rawNode) || []).map(
+        (subCategory) =>
+          `${dottedName as string} . ${subCategory as string}` as DottedName
```

Comment:
I think there is no need for the `as string` casts

Reply:
👍

Reply:
No need for DottedName either

Reply:
Forget about it, modifying this is breaking something elsewhere and I'm a lazy guy